controllers/util/storageconsumer.go

```diff
@@ -20,6 +20,13 @@ const (
 	nfsClientProfileKey             = "csiop-nfs-client-profile"
 )
 
+type AvailableServices struct {
+	Rbd    bool
+	CephFs bool
+	Nfs    bool
+	Mcg    bool
+}
+
```

Comment:
The reason to have this here instead of in storageconsumer controller is because the GetStorageConsumerDefaultResourceNames uses this struct other option would be to pass three bool for each service type

Reply:
@rewantsoni I am not sure I understand this comment. can you please provide more context?